### PR TITLE
[CCTRI-1343] Implement /health endpoint

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**
This  product returns errors with status code 200. Therefore, I used the error message to determine whether the error is critical or not critical.
Screenshot 2020-07-03 at 13 21 16
Screenshot 2020-07-03 at 13 22 09

### CODE REVIEW

File: api/utils.py [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
-    any way, replaced by another function, or even removed from the module.
-    """
-
     try:
         scheme, token = request.headers['Authorization'].split()
         assert scheme.lower() == 'bearer'
         return jwt.decode(token, current_app.config['SECRET_KEY'])
     except (KeyError, ValueError, AssertionError, JoseError):
-        raise Forbidden('Invalid Authorization Bearer JWT.')
+        raise InvalidJWTError('Invalid Authorization Bearer JWT.')
```
Comment (GeorgiyII) [inline api/utils.py:13]: Why you have two errors for one event, i mean for auth error:
InvalidJWTError and CriticalError?
You can return {} in this place and catch error from your 3-rd party)

File: api/errors.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            INVALID_ARGUMENT,
+            f'Invalid JSON payload received. {message}'
+        )
+
+
+class UnsupportedObservableTypeError(InvalidArgumentError):
+    def __init__(self, type_):
+        super().__init__(
+            f'Unsupported observable error: {type_}'
+        )
```
Comment (GeorgiyII) [inline api/errors.py:58]: You don't use this error in the code.
What is a reason of this?

Comment (AnastasiiaRozlyvan) [reply api/errors.py]: Yes, this is an extra error, thanks! Now it is deleted.

File: api/utils.py (hunk 1 above)
Comment (AlinaBlahun) [reply api/utils.py:13]: Are you suggesting doing an extra request to 3-rd party, when we are already sure that credentials are wrong?

**Pull request merged**